File: Creational/Lab1/Python/realtor.py
class Realtor:
    def __init__(self, id, name_and_surname, phone, email):
        self.id = id
        self.name_and_surname = name_and_surname
        self.phone = phone
        self.email = email

    def __str__(self):
        return f"Realtor:\nId:{self.id}, Name and surname: {self.name_and_surname},\nPhone number: {self.phone}, Email: {self.email}"

File: Structural/Lab12/C#/Entities/Realtor.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace C_.Entities
{
    public class Realtor : Person
    {
        public List<RealEstate> RealEstates {get; set;} = new List<RealEstate>();
        double YearsOfWork{ get; set; }
        public Realtor(string nameAndSurname, string phone, string email, double yearsOfWork):base(nameAndSurname, phone, email) {
            YearsOfWork = yearsOfWork;
        }

        public Realtor(Realtor realtor):base(realtor.NameAndSurname, realtor.Phone, realtor.Email) {
            this.Id =realtor.Id;
            this.RealEstates = realtor.RealEstates;
        }

        public override string ToString()
        {
            return $"Realtor:\nId:{Id}, Name and surname: {NameAndSurname},\nPhone number: {Phone}, Email: {Email}, Years of work {YearsOfWork}";
        }
    }
}
